docs/reference/run.md

```diff
@@ -505,18 +505,18 @@ parent group.
 The operator can also adjust the performance parameters of the
 container:
 
-| Option                               |  Description                                                                                      |
-|--------------------------------------|---------------------------------------------------------------------------------------------|
-| `-m`, `--memory="" `                 | Memory limit (format: , where unit = b, k, m or g)                                          |
-| `--memory-swap=""`                   | Total memory limit (memory + swap, format: , where unit = b, k, m or g)                     |
-| `-c`, `--cpu-shares=0`               | CPU shares (relative weight)                                                                |
-| `--cpu-period=0`                     | Limit the CPU CFS (Completely Fair Scheduler) period                                        |
-| `--cpuset-cpus="" `                  | CPUs in which to allow execution (0-3, 0,1)                                                 |
-| `--cpuset-mems=""`                   | Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems. |
-| `--cpu-quota=0`                      | Limit the CPU CFS (Completely Fair Scheduler) quota                                         |
-| `--blkio-weight=0`                   | Block IO weight (relative weight) accepts a weight value between 10 and 1000.               |
-| `--oom-kill-disable=true` or `false` | Whether to disable OOM Killer for the container or not.                                     |
-| `--memory-swappiness=""  `           | Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.        |
+| Option                     |  Description                                                                                   |
+|----------------------------|------------------------------------------------------------------------------------------------|
```

Comment:
@hqhq thank you for the contribution.  You need to use `tics` instead of angle brackets to show the format.  The angle brackets are not rendering properly.

![image](https://cloud.githubusercontent.com/assets/1347209/9282457/5c20fbaa-4281-11e5-8a9d-d31e8414e62c.png)